openhtf/util/data.py

```diff
@@ -32,7 +32,7 @@
 from enum import Enum
 
 # Used by convert_to_base_types().
-PASSTHROUGH_TYPES = {bool, bytes, int, long, type(None), unicode}
+PASSTHROUGH_TYPES = {bool, bytes, int, int, type(None), str}
```

Comment:
More 2to3 fun. I think we want to put `long` back, and `from past.builtins import long` at the top.

Reply:
@wallacbe, missed this?